parlai/agents/seq2seq/modules.py

```diff
@@ -434,15 +442,23 @@ def forward(self, input):
 
 class AttentionLayer(nn.Module):
     def __init__(self, attn_type, hidden_size, emb_size, bidirectional=False,
-                 attn_length=-1):
+                 attn_length=-1, attn_time='pre'):
         super().__init__()
         self.attention = attn_type
 
         if self.attention != 'none':
             hsz = hidden_size
```

Comment:
why do we rename this variable here?

Reply:
it made the lines below a lot shorter :)